src/client/pydaos/raw/daos_cref.py

```diff
@@ -160,10 +160,14 @@ class DaosObjId(ctypes.Structure):
     _fields_ = [("lo", ctypes.c_uint64),
                 ("hi", ctypes.c_uint64)]
 
+class DaosShardLoc(ctypes.Structure):
```

Comment:
(pylint-missing-docstring) Missing class docstring

Reply:
need to add 
"""description""" on first line of function to silence this